src/Libraries/Microsoft.Extensions.Compliance.Redaction/DeniedValuesAttribute.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#if !NET8_0_OR_GREATER
+
+// Stolen from dotnet/runtime/src/libraries/System.ComponentModel.Annotations/src/System/ComponentModel/DataAnnotations/DeniedValueAttribute.cs
+
+using System.Diagnostics.CodeAnalysis;
+
+namespace System.ComponentModel.DataAnnotations
+{
+    [AttributeUsage(AttributeTargets.Property | AttributeTargets.Field | AttributeTargets.Parameter,
+        AllowMultiple = false)]
+    [ExcludeFromCodeCoverage]
+    internal sealed class DeniedValuesAttribute : ValidationAttribute
```

Comment:
Is it used somewhere?

Reply:
It was at some point, but now it's dead code. It's gone